starters/cms/src/main.tsx

```diff
@@ -31,3 +24,18 @@ if (didSync) {
         </StrictMode>
     )
 }
+
+if (framer.mode === "syncManagedCollection") {
+    if (framer.isAllowedTo(...syncMethods)) {
+        const { didSync } = await syncExistingCollection(activeCollection, previousDataSourceId, previousSlugFieldId)
+        if (didSync) {
+            await framer.closePlugin("Synchronization successful", { variant: "success" })
+        } else {
+            launch()
+        }
+    } else {
+        launch()
```

Comment:
Launching UI if the permissions are off, which will look like the screenshot. Not ideal, as then you can't just sync from there, but you'll probably just close the plugin and try again later.

Reply:
`launch` isn't very descriptive by itself. How about naming it something like `showUI`?